  INFO root [2019-09-26 10:13:33,116] - 
  INFO root [2019-09-26 10:13:33,123] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:13:33,520] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:13:33,520] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:13:33,520] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:13:33,741] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:52760 )
  INFO root [2019-09-26 10:13:34,282] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:52760, SOMAXCONN=5.
  INFO root [2019-09-26 10:13:34,282] - server-configs:
  INFO root [2019-09-26 10:13:34,282] - 	internalTcpAddr : 10.15.76.77:52760
  INFO root [2019-09-26 10:13:34,282] - 	componentID : 4000
  WARN root [2019-09-26 10:13:34,283] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:13:34,518] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 8988 ----
  INFO root [2019-09-26 10:13:34,522] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:13:34,610] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:13:34,616] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:13:36,239] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:13:36,239] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 10:13:37,674] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 10:13:37,687] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:13:37,717] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 10:13:37,739] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 10:13:38,093] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
5BB2FAA1B3A0D08411976B135EA2E96C7F9A92B1C32F4DBBBFB34DB33BA12255A73E5FF0C3A4B1B8C243A31D441CEFD0F4ACC944269681460B8B2B1700753A70299BD8F6E5E4F0C5F48625AD783D4276D633833ECC999682D19A3FAEB586B924AAD551F254DDD79634F0848F2DF5E45C3C2252DE65F76D3CB82AFDE4E628F8BE
 DEBUG root [2019-09-26 10:13:38,190] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:13:40,352] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 10:13:40,352] - DBUtil::createInterface[0x1aee68]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 10:13:40,531] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:13:40,531] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 10:13:40,531] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:13:40,532] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:13:40,532] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:13:40,532] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:13:40,532] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:13:40,745] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:13:40,745] - DBThread::onStart(): 0x660cd0!
 DEBUG root [2019-09-26 10:13:40,860] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:13:40,860] - DBThread::onStart(): 0x661880!
 DEBUG root [2019-09-26 10:13:41,051] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:13:41,052] - DBThread::onStart(): 0x6613b8!
 DEBUG root [2019-09-26 10:13:41,192] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:13:41,192] - DBThread::onStart(): 0x661440!
 DEBUG root [2019-09-26 10:13:41,275] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:13:41,275] - DBThread::onStart(): 0x660c48!
  INFO root [2019-09-26 10:13:41,595] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:41,929] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:41,967] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:41,967] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:42,025] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:42,055] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:42,055] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:13:42,056] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:13:42,074] - DBTask::process(): took 0.44 seconds
sql:(delete from kbe_email_verification where logtime<1569460422 and type=3)
  WARN root [2019-09-26 10:13:42,074] - DBTask::process(): took 0.35 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  WARN root [2019-09-26 10:13:42,603] - DBTask::process(): took 0.99 seconds
sql:(show index from tbl_Account)
  WARN root [2019-09-26 10:13:42,603] - DBTask::process(): took 0.53 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-26 10:13:42,660] - DBTask::process(): took 1.04 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569464021,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569464021)
  WARN root [2019-09-26 10:13:42,664] - DBTask::process(): took 0.59 seconds
sql:(show index from tbl_Hall)
  INFO root [2019-09-26 10:13:42,726] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:13:42,762] - DBTask::process(): took 1.15 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 10:13:42,879] - DBTask::process(): delay 1.01 seconds, try adjusting the kbengine[_defs].xml(numConnections) and MySQL(my.cnf->max_connections or innodb_flush_log_at_trx_commit)!
sql:(show index from tbl_Smash)
  WARN root [2019-09-26 10:13:42,881] - DBTask::process(): delay 1.01 seconds, try adjusting the kbengine[_defs].xml(numConnections) and MySQL(my.cnf->max_connections or innodb_flush_log_at_trx_commit)!
sql:(show index from tbl_Food)
  WARN root [2019-09-26 10:13:42,881] - DBTask::process(): took 0.28 seconds
sql:(show index from tbl_Smash)
  WARN root [2019-09-26 10:13:42,882] - DBTask::process(): took 0.28 seconds
sql:(show index from tbl_Food)
  WARN root [2019-09-26 10:13:42,944] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 5.27 seconds
  INFO root [2019-09-26 10:13:43,502] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 10:13:43,503] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:13:49,684] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 10:13:49,685] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:13:49,685] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 10:13:49,690] - TCPPacketReceiver::processPendingEvents(10.15.76.77:52780): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-26 10:13:49,691] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:cellappmgr, componentID:6000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:52757, extaddr:nonsupport, extport:0,  from 10.15.76.77:52783/0/0/0.
  INFO root [2019-09-26 10:13:49,691] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:13:49,691] - Components::addComponent[cellappmgr], uid=41181, componentID=6000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:13:49,692] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:loginapp, componentID:9000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:52762, extaddr:10.15.76.77, extport:20013,  from 10.15.76.77:52786/0/0/0.
  INFO root [2019-09-26 10:13:49,692] - Components::addComponent[loginapp], uid=41181, componentID=9000, globalorderid=2, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:13:50,263] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:baseappmgr, componentID:5000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:52755, extaddr:nonsupport, extport:0,  from 10.15.76.77:52788/0/0/0.
  INFO root [2019-09-26 10:13:50,263] - Components::addComponent[baseappmgr], uid=41181, componentID=5000, globalorderid=3, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:13:50,695] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:13:50,771] - Components::findComponents: found logger, addr:10.15.76.77:52759
  INFO root [2019-09-26 10:13:50,771] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=4, grouporderid=1, totalcount=1
  WARN root [2019-09-26 10:13:50,772] - Forwarding logs to logger[10.15.76.77:52759/0/0/0]...
 ERROR root [2019-09-26 10:20:13,021] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304144974131, lastReceivedTime=3304039985613, inactivityExceptionPeriod=599489449)
  INFO root [2019-09-26 10:20:13,024] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:20:13,024] - ==> [2019-09-26 10:20:13 018] TCPPacketReceiver::processPendingEvents(10.15.76.77:52759): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:20:13,025] - ==> [2019-09-26 10:20:13 022] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304144974131, lastReceivedTime=3304039985613, inactivityExceptionPeriod=599489449)
  INFO root [2019-09-26 10:37:35,224] - 
  INFO root [2019-09-26 10:37:35,225] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:37:35,226] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:37:35,226] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:37:35,226] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:37:35,236] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:52918 )
  INFO root [2019-09-26 10:37:35,236] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:52918, SOMAXCONN=5.
  INFO root [2019-09-26 10:37:35,236] - server-configs:
  INFO root [2019-09-26 10:37:35,237] - 	internalTcpAddr : 10.15.76.77:52918
  INFO root [2019-09-26 10:37:35,237] - 	componentID : 4000
  WARN root [2019-09-26 10:37:35,237] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:37:36,742] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 17560 ----
  INFO root [2019-09-26 10:37:36,743] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:37:36,743] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:37:36,743] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:37:36,950] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:37:36,950] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 10:37:37,088] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 10:37:37,089] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:37:37,090] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 10:37:37,090] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 10:37:37,311] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
4CF88FCCECFDCE59E7D504AC74B93713C8A11C5F3FF5DF12AB4662B1DB5F8A50E5E3006D1948B91672FD6C820BB101B7B0C6B8B1ECF29B208CCC0D64EB30AA7918418123FEB34CAF5F0E55DBD2A8C305A0BF34E9A15DA1A2AEF63490DFBC6723B64F2D076CE8490A9A20EA4A45C86D78D96F5685FD9D0B484F35967C0D08C6A5
 DEBUG root [2019-09-26 10:37:37,333] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:37:37,512] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 10:37:37,513] - DBUtil::createInterface[0x22fed18]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 10:37:37,513] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:37:37,513] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 10:37:37,549] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:37:37,558] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:37:37,562] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:37:37,563] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:37:37,565] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:37:37,647] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:37:37,648] - DBThread::onStart(): 0x246a0f0!
  INFO root [2019-09-26 10:37:37,666] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 10:37:37,785] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:37:37,785] - DBThread::onStart(): 0x246a200!
 DEBUG root [2019-09-26 10:37:37,893] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:37:37,893] - DBThread::onStart(): 0x246a178!
  INFO root [2019-09-26 10:37:37,951] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:37:37,952] - DBTask::process(): took 0.28 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-26 10:37:37,984] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 10:37:38,077] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:37:38,077] - DBThread::onStart(): 0x246a288!
  WARN root [2019-09-26 10:37:38,127] - DBTask::process(): took 0.34 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569465457,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569465457)
 DEBUG root [2019-09-26 10:37:38,201] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:37:38,201] - DBThread::onStart(): 0x4b57498!
  INFO root [2019-09-26 10:37:38,259] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:37:38,301] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:37:38,302] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:37:38,302] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:37:38,302] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:37:38,303] - DBTask::process(): took 0.22 seconds
sql:(delete from kbe_email_verification where logtime<1569461858 and type=3)
  WARN root [2019-09-26 10:37:38,305] - DBTask::process(): took 0.35 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-26 10:37:38,486] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:37:38,487] - DBTask::process(): took 0.59 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 10:37:38,556] - DBTask::process(): took 0.25 seconds
sql:(show index from tbl_Smash)
  WARN root [2019-09-26 10:37:38,576] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.49 seconds
  INFO root [2019-09-26 10:37:38,999] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 10:37:39,000] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:37:45,145] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 10:37:45,145] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:37:45,146] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:37:46,149] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:37:46,221] - Components::findComponents: found logger, addr:10.15.76.77:52920
  INFO root [2019-09-26 10:37:46,221] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:37:46,222] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 10:37:46,227] - Forwarding logs to logger[10.15.76.77:52920/0/0/0]...
 ERROR root [2019-09-26 10:40:29,456] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316309319934, lastReceivedTime=3316069022738, inactivityExceptionPeriod=599492838)
  INFO root [2019-09-26 10:40:29,491] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:40:29,492] - ==> [2019-09-26 10:40:29 440] TCPPacketReceiver::processPendingEvents(10.15.76.77:52920): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:40:29,492] - ==> [2019-09-26 10:40:29 458] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316309319934, lastReceivedTime=3316069022738, inactivityExceptionPeriod=599492838)
  INFO root [2019-09-26 10:44:55,965] - 
  INFO root [2019-09-26 10:44:55,969] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:44:55,970] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:44:55,970] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:44:55,970] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:44:56,019] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53016 )
  INFO root [2019-09-26 10:44:56,021] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53016, SOMAXCONN=5.
  INFO root [2019-09-26 10:44:56,021] - server-configs:
  INFO root [2019-09-26 10:44:56,021] - 	internalTcpAddr : 10.15.76.77:53016
  INFO root [2019-09-26 10:44:56,021] - 	componentID : 4000
  WARN root [2019-09-26 10:44:56,036] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:44:57,573] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 13156 ----
  INFO root [2019-09-26 10:44:57,574] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:44:57,574] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:44:57,575] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:44:57,798] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:44:57,798] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 10:44:57,993] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 10:44:57,993] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:44:57,993] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 10:44:57,996] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 10:44:58,308] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
2D2C030352A84EF3F6179886649855D723C80051E517AF31420AA5F6403C19659EFEDCF705A710AB4791E2F5D14178DA38D42AE46447730CA8252C56017A1B2FC33191AF4040B4422D77C66321A49D2D415C924AA192CCFB821831734009411F738E5C958711446A88C46FF2B7F59A51DC9A69F0D97A1300CBB16AC1A9EA0783
 DEBUG root [2019-09-26 10:44:58,342] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:44:58,476] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 10:44:58,476] - DBUtil::createInterface[0x19ef60]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 10:44:58,476] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:44:58,477] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 10:44:58,500] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:44:58,519] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:44:58,520] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:44:58,521] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:44:58,522] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:44:58,600] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:44:58,601] - DBThread::onStart(): 0x726598!
  INFO root [2019-09-26 10:44:58,668] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 10:44:58,747] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:44:58,747] - DBThread::onStart(): 0x726c80!
 DEBUG root [2019-09-26 10:44:58,887] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:44:58,888] - DBThread::onStart(): 0x7267b8!
 DEBUG root [2019-09-26 10:44:59,038] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:44:59,038] - DBThread::onStart(): 0x726510!
 DEBUG root [2019-09-26 10:44:59,197] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:44:59,198] - DBThread::onStart(): 0x726400!
  WARN root [2019-09-26 10:44:59,292] - DBTask::process(): took 0.40 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 10:44:59,339] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:44:59,340] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:44:59,340] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:44:59,340] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:44:59,340] - DBTask::process(): took 0.67 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-26 10:44:59,341] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:44:59,341] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:44:59,342] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:44:59,342] - DBTask::process(): took 0.30 seconds
sql:(delete from kbe_email_verification where logtime<1569462299 and type=3)
  WARN root [2019-09-26 10:44:59,613] - DBTask::process(): took 0.41 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569465899,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569465899)
  WARN root [2019-09-26 10:44:59,615] - DBTask::process(): took 0.23 seconds
sql:(show index from tbl_Smash)
  INFO root [2019-09-26 10:44:59,699] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:44:59,700] - DBTask::process(): took 0.95 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 10:44:59,722] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.73 seconds
  INFO root [2019-09-26 10:45:00,186] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 10:45:00,186] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:45:06,355] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 10:45:06,355] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:45:06,356] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:45:07,373] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:45:07,444] - Components::findComponents: found logger, addr:10.15.76.77:53012
  INFO root [2019-09-26 10:45:07,444] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:45:07,445] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 10:45:07,447] - Forwarding logs to logger[10.15.76.77:53012/0/0/0]...
 ERROR root [2019-09-26 10:47:13,904] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320353796894, lastReceivedTime=3320172198692, inactivityExceptionPeriod=599463551)
  INFO root [2019-09-26 10:47:13,908] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:47:13,909] - ==> [2019-09-26 10:47:13 890] TCPPacketReceiver::processPendingEvents(10.15.76.77:53012): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:47:13,909] - ==> [2019-09-26 10:47:13 905] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320353796894, lastReceivedTime=3320172198692, inactivityExceptionPeriod=599463551)
  INFO root [2019-09-26 10:53:20,385] - 
  INFO root [2019-09-26 10:53:20,385] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:20,387] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:20,387] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:20,387] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:20,397] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53106 )
  INFO root [2019-09-26 10:53:20,398] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53106, SOMAXCONN=5.
  INFO root [2019-09-26 10:53:20,398] - server-configs:
  INFO root [2019-09-26 10:53:20,398] - 	internalTcpAddr : 10.15.76.77:53106
  INFO root [2019-09-26 10:53:20,398] - 	componentID : 4000
  WARN root [2019-09-26 10:53:20,398] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:21,904] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 12788 ----
  INFO root [2019-09-26 10:53:21,904] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:21,905] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:21,905] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:22,102] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:53:22,103] - Script::install(): KBExtra is successfully!
 ERROR root [2019-09-26 10:53:22,352] - app::initialize(): initialization failed!
  INFO root [2019-09-26 10:53:22,381] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:53:22,381] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:53:22,682] - ThreadPool::destroy(): waiting for thread(2)[0x215e418,0x215e480,], try=1
 DEBUG root [2019-09-26 10:53:23,084] - TPThread::~TPThread(): 0x215e418
 DEBUG root [2019-09-26 10:53:23,084] - TPThread::~TPThread(): 0x215e480
 DEBUG root [2019-09-26 10:53:23,084] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-26 10:53:23,084] - Bundle::destroyObjPool(): size 9.
 DEBUG root [2019-09-26 10:53:23,085] - Channel::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:23,085] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:23,085] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2019-09-26 10:53:23,085] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:23,085] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:23,086] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:23,086] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-26 10:53:29,927] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-09-26 10:53:39,019] - 
  INFO root [2019-09-26 10:53:39,021] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:39,022] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:39,022] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:39,022] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:39,071] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53129 )
  INFO root [2019-09-26 10:53:39,071] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53129, SOMAXCONN=5.
  INFO root [2019-09-26 10:53:39,071] - server-configs:
  INFO root [2019-09-26 10:53:39,071] - 	internalTcpAddr : 10.15.76.77:53129
  INFO root [2019-09-26 10:53:39,071] - 	componentID : 4000
  WARN root [2019-09-26 10:53:39,072] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:40,582] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 19068 ----
  INFO root [2019-09-26 10:53:40,583] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:40,583] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:40,584] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:40,777] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:53:40,777] - Script::install(): KBExtra is successfully!
 ERROR root [2019-09-26 10:53:40,941] - app::initialize(): initialization failed!
  INFO root [2019-09-26 10:53:40,951] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:53:40,951] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:53:41,260] - ThreadPool::destroy(): waiting for thread(2)[0x86ee28,0x86ee90,], try=1
 DEBUG root [2019-09-26 10:53:41,662] - TPThread::~TPThread(): 0x86ee28
 DEBUG root [2019-09-26 10:53:41,662] - TPThread::~TPThread(): 0x86ee90
 DEBUG root [2019-09-26 10:53:41,662] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-26 10:53:41,663] - Bundle::destroyObjPool(): size 9.
 DEBUG root [2019-09-26 10:53:41,664] - Channel::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:41,664] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:41,664] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2019-09-26 10:53:41,665] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:41,665] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:41,666] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:41,666] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-26 10:53:48,470] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-09-26 10:53:56,762] - 
  INFO root [2019-09-26 10:53:56,763] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:56,764] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:56,764] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:56,765] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:56,807] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53142 )
  INFO root [2019-09-26 10:53:56,807] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53142, SOMAXCONN=5.
  INFO root [2019-09-26 10:53:56,808] - server-configs:
  INFO root [2019-09-26 10:53:56,808] - 	internalTcpAddr : 10.15.76.77:53142
  INFO root [2019-09-26 10:53:56,808] - 	componentID : 4000
  WARN root [2019-09-26 10:53:56,809] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:58,332] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 5476 ----
  INFO root [2019-09-26 10:53:58,332] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:58,333] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:58,333] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:58,533] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:53:58,533] - Script::install(): KBExtra is successfully!
 ERROR root [2019-09-26 10:53:58,675] - app::initialize(): initialization failed!
  INFO root [2019-09-26 10:53:58,686] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:53:58,686] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:53:58,987] - ThreadPool::destroy(): waiting for thread(2)[0x216ef40,0x216efa8,], try=1
 DEBUG root [2019-09-26 10:53:59,389] - TPThread::~TPThread(): 0x216ef40
 DEBUG root [2019-09-26 10:53:59,389] - TPThread::~TPThread(): 0x216efa8
 DEBUG root [2019-09-26 10:53:59,389] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-26 10:53:59,389] - Bundle::destroyObjPool(): size 9.
 DEBUG root [2019-09-26 10:53:59,391] - Channel::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:59,391] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:59,391] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2019-09-26 10:53:59,392] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:59,392] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:59,392] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:53:59,392] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-26 10:54:06,195] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-09-26 10:54:11,921] - 
  INFO root [2019-09-26 10:54:11,923] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:54:11,925] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:54:11,925] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:54:11,925] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:54:11,936] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53172 )
  INFO root [2019-09-26 10:54:11,936] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53172, SOMAXCONN=5.
  INFO root [2019-09-26 10:54:11,936] - server-configs:
  INFO root [2019-09-26 10:54:11,936] - 	internalTcpAddr : 10.15.76.77:53172
  INFO root [2019-09-26 10:54:11,936] - 	componentID : 4000
  WARN root [2019-09-26 10:54:11,937] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:54:13,444] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 9232 ----
  INFO root [2019-09-26 10:54:13,446] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:54:13,447] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:54:13,447] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:54:13,725] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:54:13,726] - Script::install(): KBExtra is successfully!
 ERROR root [2019-09-26 10:54:13,964] - app::initialize(): initialization failed!
  INFO root [2019-09-26 10:54:13,977] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:54:13,977] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:54:14,278] - ThreadPool::destroy(): waiting for thread(2)[0xd0eb58,0xd0e950,], try=1
 DEBUG root [2019-09-26 10:54:14,680] - TPThread::~TPThread(): 0xd0eb58
 DEBUG root [2019-09-26 10:54:14,680] - TPThread::~TPThread(): 0xd0e950
 DEBUG root [2019-09-26 10:54:14,680] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-26 10:54:14,680] - Bundle::destroyObjPool(): size 9.
 DEBUG root [2019-09-26 10:54:14,680] - Channel::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:54:14,681] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:54:14,681] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2019-09-26 10:54:14,681] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:54:14,681] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:54:14,682] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:54:14,682] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-26 10:54:21,485] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-09-26 10:56:35,099] - 
  INFO root [2019-09-26 10:56:35,100] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:56:35,101] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:56:35,102] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:56:35,102] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:56:35,112] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53182 )
  INFO root [2019-09-26 10:56:35,112] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53182, SOMAXCONN=5.
  INFO root [2019-09-26 10:56:35,112] - server-configs:
  INFO root [2019-09-26 10:56:35,112] - 	internalTcpAddr : 10.15.76.77:53182
  INFO root [2019-09-26 10:56:35,112] - 	componentID : 4000
  WARN root [2019-09-26 10:56:35,113] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:56:36,621] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 16168 ----
  INFO root [2019-09-26 10:56:36,622] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:56:36,622] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:56:36,623] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:56:36,820] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:56:36,820] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 10:56:37,040] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 10:56:37,041] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:56:37,041] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 10:56:37,044] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 10:56:37,375] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
E1A50F0F36833340B633DCDCC7F6B5B0E83A1320DAC0ABDE66583578094CB7703360362AA4061F59967D5D2D7CF035BE1EC47260E2B72011F50B429F2DEA8ED5166B4D1870627ADE399E431B69B5F7EE4FEA993C9C733A1C92461740F6563F4F1D2A6C175882CEC804FC0410E79D642F2853A0D1CEF13E45AF1AC0E310C4261E
 DEBUG root [2019-09-26 10:56:37,379] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:56:37,706] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 10:56:37,707] - DBUtil::createInterface[0x7ceff4]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 10:56:37,707] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:56:37,707] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 10:56:37,713] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:56:37,719] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:56:37,720] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:56:37,721] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:56:37,730] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 10:56:37,890] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:56:37,891] - DBThread::onStart(): 0xc01ac0!
 DEBUG root [2019-09-26 10:56:37,958] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:56:37,959] - DBThread::onStart(): 0xc01b48!
 DEBUG root [2019-09-26 10:56:38,059] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:56:38,059] - DBThread::onStart(): 0xc01460!
 DEBUG root [2019-09-26 10:56:38,117] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:56:38,117] - DBThread::onStart(): 0xc01c58!
  INFO root [2019-09-26 10:56:38,188] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 10:56:38,258] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 10:56:38,258] - DBThread::onStart(): 0xc02098!
  INFO root [2019-09-26 10:56:38,349] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:56:38,349] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:56:38,350] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:56:38,350] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:56:38,419] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:56:38,419] - DBTask::process(): took 0.23 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 10:56:38,419] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 10:56:38,420] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:56:38,420] - DBTask::process(): took 0.23 seconds
sql:(delete from kbe_email_verification where logtime<1569462998 and type=3)
  WARN root [2019-09-26 10:56:38,597] - DBTask::process(): took 0.41 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569466598,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569466598)
  INFO root [2019-09-26 10:56:38,689] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 10:56:38,690] - DBTask::process(): took 0.43 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 10:56:38,699] - DBTask::process(): took 0.27 seconds
sql:(show index from tbl_Hall)
  WARN root [2019-09-26 10:56:38,699] - DBTask::process(): took 0.27 seconds
sql:(show index from tbl_Food)
  WARN root [2019-09-26 10:56:38,700] - DBTask::process(): took 0.27 seconds
sql:(show index from tbl_Smash)
  WARN root [2019-09-26 10:56:38,717] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.68 seconds
  INFO root [2019-09-26 10:56:39,138] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 10:56:39,139] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:56:45,285] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 10:56:45,286] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:56:45,286] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:56:46,301] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:56:46,393] - Components::findComponents: found logger, addr:10.15.76.77:53185
  INFO root [2019-09-26 10:56:46,394] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:56:46,394] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 10:56:46,395] - Forwarding logs to logger[10.15.76.77:53185/0/0/0]...
 ERROR root [2019-09-26 10:58:37,840] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327193160793, lastReceivedTime=3327166853366, inactivityExceptionPeriod=599493670)
  INFO root [2019-09-26 10:58:37,848] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:58:37,848] - ==> [2019-09-26 10:58:37 831] TCPPacketReceiver::processPendingEvents(10.15.76.77:53185): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:58:37,848] - ==> [2019-09-26 10:58:37 841] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327193160793, lastReceivedTime=3327166853366, inactivityExceptionPeriod=599493670)
  INFO root [2019-09-26 11:00:58,112] - 
  INFO root [2019-09-26 11:00:58,114] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:00:58,115] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:00:58,115] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:00:58,115] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:00:58,140] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53254 )
  INFO root [2019-09-26 11:00:58,140] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53254, SOMAXCONN=5.
  INFO root [2019-09-26 11:00:58,141] - server-configs:
  INFO root [2019-09-26 11:00:58,141] - 	internalTcpAddr : 10.15.76.77:53254
  INFO root [2019-09-26 11:00:58,141] - 	componentID : 4000
  WARN root [2019-09-26 11:00:58,142] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:00:59,649] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 15640 ----
  INFO root [2019-09-26 11:00:59,650] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:00:59,650] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:00:59,651] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:00:59,855] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:00:59,855] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:01:00,267] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:01:00,267] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:01:00,268] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:01:00,277] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:01:00,606] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
0D0B30615971AE7732C436D7746717801C63FCC3EE6EEE79D7E6049D012853B31241B02459B5909F7415E502F8031F3FBBEBB8B0594F3D24DDF184FA05840765D8337C056E1B8C3F7EAFBB27C77DE01DA28AE9796513D7E56F91E7E8D60FCCD8A97231E9C8B051DD56D7727D57C4DC5802D47D0E1F30F581EAD9E9F24818FE2A
 DEBUG root [2019-09-26 11:01:00,647] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:01:00,810] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:01:00,811] - DBUtil::createInterface[0xafef84]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:01:00,811] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:01:00,811] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:01:00,833] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:01:00,841] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:01:00,856] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:01:00,858] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:01:00,863] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:01:00,933] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:01:00,933] - DBThread::onStart(): 0xe9bdf0!
  INFO root [2019-09-26 11:01:00,935] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:01:01,110] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:01:01,110] - DBThread::onStart(): 0xe9bb48!
 DEBUG root [2019-09-26 11:01:01,212] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:01:01,213] - DBThread::onStart(): 0xe9be78!
 DEBUG root [2019-09-26 11:01:01,320] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:01:01,320] - DBThread::onStart(): 0xe9bc58!
  INFO root [2019-09-26 11:01:01,460] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:01:01,518] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:01:01,518] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:01:01,519] - DBTask::process(): took 0.58 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-26 11:01:01,519] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:01:01,520] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:01:01,520] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:01:01,521] - DBTask::process(): took 0.31 seconds
sql:(delete from kbe_email_verification where logtime<1569463261 and type=3)
 DEBUG root [2019-09-26 11:01:01,670] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:01:01,670] - DBThread::onStart(): 0xe9b0a8!
  WARN root [2019-09-26 11:01:01,701] - DBTask::process(): took 0.59 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569466861,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569466861)
  WARN root [2019-09-26 11:01:01,829] - DBTask::process(): took 0.51 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 11:01:01,860] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:01:01,862] - DBTask::process(): took 0.34 seconds
sql:(show index from tbl_Hall)
  WARN root [2019-09-26 11:01:01,862] - DBTask::process(): took 0.34 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-26 11:01:01,961] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:01:01,962] - DBTask::process(): took 0.29 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:01:01,978] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.71 seconds
  INFO root [2019-09-26 11:01:02,397] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:01:02,397] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:01:08,545] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:01:08,546] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:01:08,546] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:01:09,548] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:01:09,636] - Components::findComponents: found logger, addr:10.15.76.77:53251
  INFO root [2019-09-26 11:01:09,636] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:01:09,637] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 11:01:09,638] - Forwarding logs to logger[10.15.76.77:53251/0/0/0]...
 ERROR root [2019-09-26 11:02:47,355] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329688310728, lastReceivedTime=3329495439977, inactivityExceptionPeriod=599489321)
  INFO root [2019-09-26 11:02:47,361] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:02:47,361] - ==> [2019-09-26 11:02:47 353] TCPPacketReceiver::processPendingEvents(10.15.76.77:53251): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:02:47,361] - ==> [2019-09-26 11:02:47 356] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329688310728, lastReceivedTime=3329495439977, inactivityExceptionPeriod=599489321)
  INFO root [2019-09-26 11:02:49,541] - 
  INFO root [2019-09-26 11:02:49,548] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:02:49,550] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:02:49,550] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:02:49,550] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:02:49,623] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53309 )
  INFO root [2019-09-26 11:02:49,623] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53309, SOMAXCONN=5.
  INFO root [2019-09-26 11:02:49,623] - server-configs:
  INFO root [2019-09-26 11:02:49,623] - 	internalTcpAddr : 10.15.76.77:53309
  INFO root [2019-09-26 11:02:49,623] - 	componentID : 4000
  WARN root [2019-09-26 11:02:49,636] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:02:51,142] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 6892 ----
  INFO root [2019-09-26 11:02:51,143] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:02:51,143] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:02:51,143] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:02:51,392] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:02:51,392] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:02:51,684] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:02:51,684] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:02:51,685] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:02:51,687] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:02:51,984] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
0E0D3EC4170FD4196296D1303F57D45264C2461F8FF5A6C4E4123AA68A61C3BC313F53CACE2D7B924180C3B81EE69320EC6AEE6AE616922ECC12AA1AFA0DCE16B62C02DEC3ABC811892846B6C7833322A719AA896A50119B6BAD5D8767CA44576B53DE8DF29044CD8CCCD027B95DF56EE099653542655A015E352E9D5B15A6C7
 DEBUG root [2019-09-26 11:02:51,990] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:02:52,226] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:02:52,227] - DBUtil::createInterface[0xeff24c]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:02:52,227] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:02:52,228] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:02:52,233] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:02:52,251] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:02:52,252] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:02:52,265] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:02:52,270] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:02:52,353] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:02:52,354] - DBThread::onStart(): 0x21fb970!
  INFO root [2019-09-26 11:02:52,355] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:02:52,444] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:02:52,444] - DBThread::onStart(): 0x21fbb08!
 DEBUG root [2019-09-26 11:02:52,503] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:02:52,503] - DBThread::onStart(): 0x21fbc18!
  INFO root [2019-09-26 11:02:52,552] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:02:52,552] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:02:52,603] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:02:52,603] - DBThread::onStart(): 0x21fbe38!
 DEBUG root [2019-09-26 11:02:52,710] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:02:52,710] - DBThread::onStart(): 0x21fbca0!
  WARN root [2019-09-26 11:02:52,779] - DBTask::process(): took 0.23 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-26 11:02:52,804] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:02:52,804] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:02:52,804] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:02:52,805] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:02:52,805] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:02:52,805] - DBTask::process(): took 0.36 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  INFO root [2019-09-26 11:02:52,806] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:02:52,807] - DBTask::process(): took 0.30 seconds
sql:(delete from kbe_email_verification where logtime<1569463372 and type=3)
  WARN root [2019-09-26 11:02:52,911] - DBTask::process(): took 0.31 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569466972,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569466972)
  WARN root [2019-09-26 11:02:52,961] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.28 seconds
  INFO root [2019-09-26 11:02:53,429] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:02:53,429] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:02:59,705] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:02:59,705] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:02:59,705] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 11:02:59,706] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53327): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-26 11:03:00,724] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:03:00,774] - Components::findComponents: found logger, addr:10.15.76.77:53306
  INFO root [2019-09-26 11:03:00,774] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:03:00,774] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:03:00,785] - Components::process(): Found all the components!
  WARN root [2019-09-26 11:03:00,801] - Forwarding logs to logger[10.15.76.77:53306/0/0/0]...
 ERROR root [2019-09-26 11:04:12,359] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330538354454, lastReceivedTime=3330309279056, inactivityExceptionPeriod=599483055)
  INFO root [2019-09-26 11:04:12,363] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:04:12,363] - ==> [2019-09-26 11:04:12 357] TCPPacketReceiver::processPendingEvents(10.15.76.77:53306): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:04:12,364] - ==> [2019-09-26 11:04:12 360] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330538354454, lastReceivedTime=3330309279056, inactivityExceptionPeriod=599483055)
  INFO root [2019-09-26 11:06:52,620] - 
  INFO root [2019-09-26 11:06:52,620] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:06:52,621] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:06:52,621] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:06:52,621] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:06:52,633] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53381 )
  INFO root [2019-09-26 11:06:52,633] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53381, SOMAXCONN=5.
  INFO root [2019-09-26 11:06:52,633] - server-configs:
  INFO root [2019-09-26 11:06:52,634] - 	internalTcpAddr : 10.15.76.77:53381
  INFO root [2019-09-26 11:06:52,634] - 	componentID : 4000
  WARN root [2019-09-26 11:06:52,634] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:06:54,140] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 14924 ----
  INFO root [2019-09-26 11:06:54,141] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:06:54,141] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:06:54,142] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:06:54,345] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:06:54,345] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:06:54,488] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:06:54,489] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:06:54,489] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:06:54,490] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:06:54,795] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
533E30165D6F84C009881A571EFA5FE7EDD21AFC597C3D21300C177F51718F7D406591D9CFACEAA7440D1535BE8BB0CD88083004DF8F5DA263DAC242E0B255874662B10E7ADB62B7407065B42AE88B85D49684A79E1D213BCE0BAF3FA2B83ED7D8087EBFA762DC69C8C056FADF3CADB5D3404FE7C363B8997D99A7E843C2EF94
 DEBUG root [2019-09-26 11:06:54,809] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:06:55,015] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:06:55,015] - DBUtil::createInterface[0x57ed94]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:06:55,015] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:06:55,016] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:06:55,021] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:06:55,030] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:06:55,038] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:06:55,045] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:06:55,050] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:06:55,120] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:06:55,121] - DBThread::onStart(): 0x6dbb30!
  INFO root [2019-09-26 11:06:55,122] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:06:55,212] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:06:55,212] - DBThread::onStart(): 0x6db008!
 DEBUG root [2019-09-26 11:06:55,288] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:06:55,288] - DBThread::onStart(): 0x6db998!
  INFO root [2019-09-26 11:06:55,370] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:06:55,370] - DBTask::process(): took 0.25 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
 DEBUG root [2019-09-26 11:06:55,437] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:06:55,438] - DBThread::onStart(): 0x6dbaa8!
 DEBUG root [2019-09-26 11:06:55,528] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:06:55,528] - DBThread::onStart(): 0x6daf80!
  WARN root [2019-09-26 11:06:55,530] - DBTask::process(): took 0.32 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 11:06:55,586] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:06:55,588] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:06:55,588] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:06:55,589] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:06:55,620] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:06:55,621] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:06:55,624] - DBTask::process(): took 0.25 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-26 11:06:55,770] - DBTask::process(): took 0.48 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569467215,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569467215)
  INFO root [2019-09-26 11:06:55,814] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:06:55,815] - DBTask::process(): took 0.29 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:06:55,838] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.35 seconds
  INFO root [2019-09-26 11:06:56,289] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:06:56,290] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:07:02,452] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:07:02,453] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:07:02,453] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:07:03,469] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:07:03,500] - Components::findComponents: found logger, addr:10.15.76.77:53383
  INFO root [2019-09-26 11:07:03,500] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:07:03,500] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:07:03,523] - Components::process(): Found all the components!
  WARN root [2019-09-26 11:07:03,524] - Forwarding logs to logger[10.15.76.77:53383/0/0/0]...
 ERROR root [2019-09-26 11:08:49,377] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333308522306, lastReceivedTime=3333041706058, inactivityExceptionPeriod=599486827)
  INFO root [2019-09-26 11:08:49,389] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:08:49,390] - ==> [2019-09-26 11:08:49 373] TCPPacketReceiver::processPendingEvents(10.15.76.77:53383): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:08:49,390] - ==> [2019-09-26 11:08:49 377] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333308522306, lastReceivedTime=3333041706058, inactivityExceptionPeriod=599486827)
  INFO root [2019-09-26 11:08:51,339] - 
  INFO root [2019-09-26 11:08:51,340] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:08:51,341] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:08:51,341] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:08:51,341] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:08:51,353] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53454 )
  INFO root [2019-09-26 11:08:51,354] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53454, SOMAXCONN=5.
  INFO root [2019-09-26 11:08:51,354] - server-configs:
  INFO root [2019-09-26 11:08:51,354] - 	internalTcpAddr : 10.15.76.77:53454
  INFO root [2019-09-26 11:08:51,354] - 	componentID : 4000
  WARN root [2019-09-26 11:08:51,355] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:08:52,860] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 10952 ----
  INFO root [2019-09-26 11:08:52,860] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:08:52,861] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:08:52,861] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:08:53,063] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:08:53,064] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:08:53,214] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:08:53,215] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:08:53,216] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:08:53,217] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:08:53,502] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
EA28B3FBD8DB171D7B51809022CF098DA3A8E46B85422A65246C0E7DCF96495BCC8011A943E958E33E813F36FC537DB9C7E4725A2962CE1FF4004166A0523A48C59523EF3B2CDD9FB41EFD3F6F2B4FEEB513C45C43C4CC51FAFFE61C4E505ABE851DCDADC3EDD3658257318477B4BC200EDC58EFA5EE620E7C9D28520EFCA229
 DEBUG root [2019-09-26 11:08:53,507] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:08:53,646] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:08:53,646] - DBUtil::createInterface[0xf5f41c]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:08:53,646] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:08:53,647] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:08:53,647] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:08:53,647] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:08:53,647] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:08:53,648] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:08:53,648] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:08:53,750] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:08:53,750] - DBThread::onStart(): 0x22d64e0!
 DEBUG root [2019-09-26 11:08:53,838] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:08:53,838] - DBThread::onStart(): 0x22d6678!
 DEBUG root [2019-09-26 11:08:53,938] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:08:53,938] - DBThread::onStart(): 0x22d6bc8!
 DEBUG root [2019-09-26 11:08:54,029] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:08:54,030] - DBThread::onStart(): 0x22d65f0!
  INFO root [2019-09-26 11:08:54,110] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:08:54,180] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:08:54,180] - DBThread::onStart(): 0x22d7008!
  INFO root [2019-09-26 11:08:54,238] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:08:54,279] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:08:54,279] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:08:54,279] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:08:54,341] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:08:54,342] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:08:54,342] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:08:54,343] - DBTask::process(): took 0.23 seconds
sql:(delete from kbe_email_verification where logtime<1569463734 and type=3)
  WARN root [2019-09-26 11:08:54,379] - DBTask::process(): took 0.27 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 11:08:54,412] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:08:54,413] - DBTask::process(): took 0.23 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:08:54,505] - DBTask::process(): took 0.39 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569467334,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569467334)
  WARN root [2019-09-26 11:08:54,573] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.36 seconds
  INFO root [2019-09-26 11:08:55,030] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:08:55,031] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:09:01,196] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:09:01,196] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:09:01,197] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:09:02,215] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:09:02,225] - Components::findComponents: found logger, addr:10.15.76.77:53452
  INFO root [2019-09-26 11:09:02,225] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:09:02,225] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:09:02,235] - Components::process(): Found all the components!
  WARN root [2019-09-26 11:09:02,242] - Forwarding logs to logger[10.15.76.77:53452/0/0/0]...
 ERROR root [2019-09-26 11:10:11,694] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334131691862, lastReceivedTime=3333927991510, inactivityExceptionPeriod=599492838)
  INFO root [2019-09-26 11:10:11,702] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:10:11,702] - ==> [2019-09-26 11:10:11 690] TCPPacketReceiver::processPendingEvents(10.15.76.77:53452): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:10:11,702] - ==> [2019-09-26 11:10:11 694] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334131691862, lastReceivedTime=3333927991510, inactivityExceptionPeriod=599492838)
  INFO root [2019-09-26 11:10:13,593] - 
  INFO root [2019-09-26 11:10:13,596] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:10:13,597] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:10:13,597] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:10:13,597] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:10:13,637] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53493 )
  INFO root [2019-09-26 11:10:13,638] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53493, SOMAXCONN=5.
  INFO root [2019-09-26 11:10:13,638] - server-configs:
  INFO root [2019-09-26 11:10:13,638] - 	internalTcpAddr : 10.15.76.77:53493
  INFO root [2019-09-26 11:10:13,638] - 	componentID : 4000
  WARN root [2019-09-26 11:10:13,650] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:10:15,162] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 9128 ----
  INFO root [2019-09-26 11:10:15,163] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:10:15,163] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:10:15,164] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:10:15,361] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:10:15,361] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:10:15,528] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:10:15,529] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:10:15,529] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:10:15,533] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:10:15,774] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
C81C57574BBA1AE7CA324F08A20F24752186F3A7E85AA18AD5C84BDE74CBEFE29382320F1A991A942BEDAA4B2197284F923295CF2A3F8DAE47F372938C918F9F254BB8EC17FA5FEE43DD6D0FFD093024F90BB0F488141502B6FB484EE832C2C1EC6070C3D982BF381B23A11CA03731245CCAA553E3A336442BFAD39ED55EBB68
 DEBUG root [2019-09-26 11:10:15,778] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:10:15,909] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:10:15,909] - DBUtil::createInterface[0xf2f204]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:10:15,909] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:10:15,910] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:10:15,911] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:10:15,913] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:10:15,913] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:10:15,913] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:10:15,929] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:10:16,013] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:10:16,014] - DBThread::onStart(): 0x24bd050!
  INFO root [2019-09-26 11:10:16,015] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:10:16,121] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:10:16,121] - DBThread::onStart(): 0x24bcf40!
  INFO root [2019-09-26 11:10:16,179] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:10:16,258] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:10:16,258] - DBThread::onStart(): 0x24bd0d8!
 DEBUG root [2019-09-26 11:10:16,313] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:10:16,313] - DBThread::onStart(): 0x24bcfc8!
 DEBUG root [2019-09-26 11:10:16,372] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:10:16,372] - DBThread::onStart(): 0x24bd848!
  WARN root [2019-09-26 11:10:16,432] - DBTask::process(): took 0.25 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-26 11:10:16,455] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:10:16,455] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:10:16,455] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:10:16,456] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:10:16,456] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:10:16,456] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:10:16,457] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:10:16,497] - DBTask::process(): took 0.37 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:10:16,638] - DBTask::process(): took 0.32 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569467416,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569467416)
  WARN root [2019-09-26 11:10:16,662] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.13 seconds
  INFO root [2019-09-26 11:10:17,104] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:10:17,104] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:10:23,246] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:10:23,246] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:10:23,246] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:10:24,248] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:10:24,266] - Components::findComponents: found logger, addr:10.15.76.77:53490
  INFO root [2019-09-26 11:10:24,266] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:10:24,266] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:10:24,288] - Components::process(): Found all the components!
  WARN root [2019-09-26 11:10:24,305] - Forwarding logs to logger[10.15.76.77:53490/0/0/0]...
 ERROR root [2019-09-26 11:12:29,803] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335512788230, lastReceivedTime=3335349688497, inactivityExceptionPeriod=599435862)
  INFO root [2019-09-26 11:12:29,805] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:12:29,805] - ==> [2019-09-26 11:12:29 802] TCPPacketReceiver::processPendingEvents(10.15.76.77:53490): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:12:29,805] - ==> [2019-09-26 11:12:29 804] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335512788230, lastReceivedTime=3335349688497, inactivityExceptionPeriod=599435862)
  INFO root [2019-09-26 11:12:31,496] - 
  INFO root [2019-09-26 11:12:31,497] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:12:31,498] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:12:31,498] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:12:31,498] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:12:31,559] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53566 )
  INFO root [2019-09-26 11:12:31,559] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53566, SOMAXCONN=5.
  INFO root [2019-09-26 11:12:31,560] - server-configs:
  INFO root [2019-09-26 11:12:31,560] - 	internalTcpAddr : 10.15.76.77:53566
  INFO root [2019-09-26 11:12:31,560] - 	componentID : 4000
  WARN root [2019-09-26 11:12:31,562] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:12:33,081] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 20444 ----
  INFO root [2019-09-26 11:12:33,081] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:12:33,082] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:12:33,082] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:12:33,277] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:12:33,277] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:12:33,407] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:12:33,408] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:12:33,408] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:12:33,408] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:12:33,649] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
E3F4326655E43C84D327E4C75524AEBE6CFD67B8E04560FE7EDC143E1ADDF8642AAD929A592AE275AC95CC24F3008FFBCB7552C4D9EBDDB383B7A55B4DB7A954E4D31F8E3787938C9894EE6D3958308C4823B5CF221481AA43F557207D0E48E06385579ED14723BF2EFAD5ADE812BAE13B4F0671355886EC3A6E4E47C130120E
 DEBUG root [2019-09-26 11:12:33,654] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:12:33,868] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:12:33,868] - DBUtil::createInterface[0x93ee50]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:12:33,868] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:12:33,869] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:12:33,871] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:12:33,871] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:12:33,888] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:12:33,889] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:12:33,896] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:12:34,002] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:12:34,002] - DBThread::onStart(): 0xa7fb00!
 DEBUG root [2019-09-26 11:12:34,072] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:12:34,072] - DBThread::onStart(): 0xa80160!
  INFO root [2019-09-26 11:12:34,074] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:12:34,116] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:12:34,116] - DBThread::onStart(): 0xa805a0!
 DEBUG root [2019-09-26 11:12:34,181] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:12:34,181] - DBThread::onStart(): 0xa80958!
 DEBUG root [2019-09-26 11:12:34,240] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:12:34,240] - DBThread::onStart(): 0xa7ff40!
  INFO root [2019-09-26 11:12:34,374] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:12:34,413] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:12:34,414] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:12:34,414] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:12:34,414] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:12:34,415] - DBTask::process(): took 0.34 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-26 11:12:34,415] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:12:34,416] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:12:34,417] - DBTask::process(): took 0.30 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 11:12:34,498] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:12:34,499] - DBTask::process(): took 0.42 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:12:34,616] - DBTask::process(): took 0.43 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569467554,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569467554)
  WARN root [2019-09-26 11:12:34,700] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.29 seconds
  INFO root [2019-09-26 11:12:35,130] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:12:35,131] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:12:41,299] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:12:41,299] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:12:41,300] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:12:42,303] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:12:42,355] - Components::findComponents: found logger, addr:10.15.76.77:53572
  INFO root [2019-09-26 11:12:42,355] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:12:42,356] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 11:12:42,357] - Forwarding logs to logger[10.15.76.77:53572/0/0/0]...
 ERROR root [2019-09-26 11:15:24,610] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337260858589, lastReceivedTime=3337030903570, inactivityExceptionPeriod=599501279)
  INFO root [2019-09-26 11:15:24,615] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:15:24,615] - ==> [2019-09-26 11:15:24 607] TCPPacketReceiver::processPendingEvents(10.15.76.77:53572): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:15:24,616] - ==> [2019-09-26 11:15:24 611] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337260858589, lastReceivedTime=3337030903570, inactivityExceptionPeriod=599501279)
  INFO root [2019-09-26 11:15:26,407] - 
  INFO root [2019-09-26 11:15:26,407] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:15:26,409] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:15:26,409] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:15:26,409] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:15:26,419] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53629 )
  INFO root [2019-09-26 11:15:26,419] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53629, SOMAXCONN=5.
  INFO root [2019-09-26 11:15:26,419] - server-configs:
  INFO root [2019-09-26 11:15:26,419] - 	internalTcpAddr : 10.15.76.77:53629
  INFO root [2019-09-26 11:15:26,419] - 	componentID : 4000
  WARN root [2019-09-26 11:15:26,420] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:15:27,926] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 2656 ----
  INFO root [2019-09-26 11:15:27,927] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:15:27,927] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:15:27,928] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:15:28,128] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:15:28,129] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:15:28,270] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:15:28,271] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:15:28,271] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:15:28,271] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:15:28,495] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
ED037AEAF86909747816CE10F39ED9C13B4AF6FB5D16D641DF32D9C5CD4BBB9C1EDD8C8DB93A726D948F362F734745B1505AF144B3334D06C037EE405D86ED6A099C9A425CA860E16D6AE223671AF1F956DE3E58963C415F77CEDA68D7F707FB85503CF0398276D78CF30FDA98744D6E6D5DC6C45056BAAF1EF6DE589ECD9086
 DEBUG root [2019-09-26 11:15:28,500] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:15:28,638] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:15:28,638] - DBUtil::createInterface[0xdbf024]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:15:28,638] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:15:28,639] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:15:28,639] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:15:28,639] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:15:28,639] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:15:28,640] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:15:28,640] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:15:28,778] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:15:28,779] - DBThread::onStart(): 0x22ebb48!
 DEBUG root [2019-09-26 11:15:28,941] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:15:28,941] - DBThread::onStart(): 0x22ebc58!
  INFO root [2019-09-26 11:15:28,943] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:15:29,132] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:15:29,132] - DBThread::onStart(): 0x22ebce0!
 DEBUG root [2019-09-26 11:15:29,215] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:15:29,215] - DBThread::onStart(): 0x22ebac0!
 DEBUG root [2019-09-26 11:15:29,274] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:15:29,274] - DBThread::onStart(): 0x22eb2c8!
  INFO root [2019-09-26 11:15:29,332] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:15:29,357] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:15:29,357] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:15:29,357] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:15:29,357] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:15:29,358] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:15:29,358] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:15:29,358] - DBTask::process(): took 0.23 seconds
sql:(delete from kbe_email_verification where logtime<1569464129 and type=3)
  WARN root [2019-09-26 11:15:29,528] - DBTask::process(): took 0.58 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569467729,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569467729)
  INFO root [2019-09-26 11:15:29,603] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:15:29,604] - DBTask::process(): took 0.66 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:15:29,605] - DBTask::process(): took 0.21 seconds
sql:(show index from tbl_Smash)
  WARN root [2019-09-26 11:15:29,622] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.35 seconds
  INFO root [2019-09-26 11:15:30,034] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:15:30,035] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:15:36,168] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:15:36,168] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:15:36,168] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:15:37,171] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:15:37,242] - Components::findComponents: found logger, addr:10.15.76.77:53628
  INFO root [2019-09-26 11:15:37,242] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:15:37,242] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 11:15:37,244] - Forwarding logs to logger[10.15.76.77:53628/0/0/0]...
 ERROR root [2019-09-26 11:18:22,250] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339037254149, lastReceivedTime=3338778395852, inactivityExceptionPeriod=599492902)
  INFO root [2019-09-26 11:18:22,259] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:18:22,260] - ==> [2019-09-26 11:18:22 246] TCPPacketReceiver::processPendingEvents(10.15.76.77:53628): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:18:22,261] - ==> [2019-09-26 11:18:22 250] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339037254149, lastReceivedTime=3338778395852, inactivityExceptionPeriod=599492902)
  INFO root [2019-09-26 11:19:32,948] - 
  INFO root [2019-09-26 11:19:32,949] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:19:32,950] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:19:32,951] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:19:32,951] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:19:32,962] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53701 )
  INFO root [2019-09-26 11:19:32,962] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53701, SOMAXCONN=5.
  INFO root [2019-09-26 11:19:32,962] - server-configs:
  INFO root [2019-09-26 11:19:32,962] - 	internalTcpAddr : 10.15.76.77:53701
  INFO root [2019-09-26 11:19:32,962] - 	componentID : 4000
  WARN root [2019-09-26 11:19:32,963] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:19:34,469] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 15256 ----
  INFO root [2019-09-26 11:19:34,469] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:19:34,469] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:19:34,470] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:19:34,691] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:19:34,691] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-26 11:19:34,848] - Dbmgr::initializeEnd: digest(05823D592B2521155E5056B6342AFBAA)
S_INFO root [2019-09-26 11:19:34,849] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:19:34,849] - TelnetServer server is running on port 32000
  INFO root [2019-09-26 11:19:34,851] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-26 11:19:35,117] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
4B74DB58A70C63085494FF0B28953A156FD454DB36D4A02889C8CAA7B6E33CBAEE84248BFA396C20887A4589A3DA5552205F36A37C1F69ED975C0D0302C92ADCF70DBDFB6A608B6B06517E4F5E76BC44265A07D40E70B9D7906633E8223E0D5BEB2EFD140A293AEAEB7CF9EDFE2EA12ABF10545C4F4CAA71722285AE14F6287F
 DEBUG root [2019-09-26 11:19:35,121] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:19:35,311] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-26 11:19:35,312] - DBUtil::createInterface[0x1eefa4]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-26 11:19:35,312] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:19:35,313] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-26 11:19:35,313] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:19:35,313] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:19:35,313] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:19:35,314] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:19:35,325] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-26 11:19:35,399] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:19:35,399] - DBThread::onStart(): 0x5db9a8!
 DEBUG root [2019-09-26 11:19:35,491] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:19:35,491] - DBThread::onStart(): 0x5db3d0!
 DEBUG root [2019-09-26 11:19:35,575] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:19:35,576] - DBThread::onStart(): 0x5db700!
  INFO root [2019-09-26 11:19:35,672] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-26 11:19:35,734] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:19:35,734] - DBThread::onStart(): 0x5db788!
 DEBUG root [2019-09-26 11:19:35,816] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-26 11:19:35,816] - DBThread::onStart(): 0x5db810!
  INFO root [2019-09-26 11:19:35,874] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:19:35,900] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:19:35,900] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:19:35,900] - DBTask::process(): took 0.23 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-26 11:19:35,901] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:19:35,944] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:19:35,944] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-26 11:19:35,945] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:19:35,945] - DBTask::process(): took 0.21 seconds
sql:(delete from kbe_email_verification where logtime<1569464375 and type=3)
  WARN root [2019-09-26 11:19:36,028] - DBTask::process(): took 0.35 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569467975,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569467975)
  WARN root [2019-09-26 11:19:36,089] - DBTask::process(): took 0.27 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-26 11:19:36,149] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-26 11:19:36,150] - DBTask::process(): took 0.48 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-26 11:19:36,200] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.35 seconds
  INFO root [2019-09-26 11:19:36,643] - ---- dbmgr is running ----
 DEBUG root [2019-09-26 11:19:36,643] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:19:42,811] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.76.77, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-26 11:19:42,811] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:19:42,812] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:19:43,829] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:19:43,929] - Components::findComponents: found logger, addr:10.15.76.77:53696
  INFO root [2019-09-26 11:19:43,929] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:19:43,929] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-26 11:19:43,931] - Forwarding logs to logger[10.15.76.77:53696/0/0/0]...
 ERROR root [2019-09-26 11:21:25,810] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340872863839, lastReceivedTime=3340642843715, inactivityExceptionPeriod=599494373)
  INFO root [2019-09-26 11:21:25,812] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:21:25,812] - ==> [2019-09-26 11:21:25 807] TCPPacketReceiver::processPendingEvents(10.15.76.77:53696): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:21:25,813] - ==> [2019-09-26 11:21:25 811] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340872863839, lastReceivedTime=3340642843715, inactivityExceptionPeriod=599494373)
